code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -137,6 +137,10 @@
 	/// Whether or not this plant is ghost possessable
 	var/playable_plant = TRUE
 
+/mob/living/simple_animal/hostile/venus_human_trap/Initialize(mapload)
+	.=..()
+	unique_name = TRUE
+
```

Comment:
This isn't working because you're setting unique_name to `TRUE` AFTER the rest of `Initialize()` has been called via the super. The code that runs if `unique_name` is `TRUE` runs in the parent proc, so when it's ran `unique_name` is `FALSE` still.

If you set unique_name to true in the define for the mob, it SHOULD work as intended. 

![image](https://user-images.githubusercontent.com/51863163/124660676-23991000-de6c-11eb-9b99-ef4111515bbb.png)

![image](https://user-images.githubusercontent.com/51863163/124660664-1f6cf280-de6c-11eb-8099-5551bde03ca5.png)

Reply:
Thank you for the help! It should be fixed now